### DISCUSSION TITLE
KDL-49WD751

### DISCUSSION THREAD

**Discussion by ghost** [custom-components/media_player.braviatv_psk#65] [Q&A]
Hi,

I'm trying integrate my tv into HA using your custom component.

After entering the host ip address but before entering the pin, this is written to the log ...
Exception: HTTPConnectionPool(host='192.168.1.65', port=80): Max retries exceeded with url: /sony/accessControl (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6c546340>: Failed to establish a new connection: [Errno 111] Connection refused'))

Using both Telnet and Test-NetConnection I am unable to open port 80 on my tv. Does this mean my tv (KDL-49WD751) doesn't offer the interface required by the custom component?

Any and all help gratefully received.

**Comment by gerard33** [comment]
This integration doesn't work with a PIN but with a PSK which you need to configure in the TV settings beforehand.
More details can be found in the [instructions](https://github.com/custom-components/media_player.braviatv_psk#installation-instructions-tv).
